Marlin/configuration_store.cpp

```diff
@@ -343,22 +362,15 @@ void MarlinSettings::postprocess() {
         "MBL Z array is the wrong size."
       );
       const bool leveling_is_on = TEST(mbl.status, MBL_STATUS_HAS_MESH_BIT);
-      const uint8_t mesh_num_x = GRID_MAX_POINTS_X, mesh_num_y = GRID_MAX_POINTS_Y;
       EEPROM_WRITE(leveling_is_on);
       EEPROM_WRITE(mbl.z_offset);
-      EEPROM_WRITE(mesh_num_x);
-      EEPROM_WRITE(mesh_num_y);
```

Comment:
This should be left as-is, along with the loop writing `dummy` below. The code that sanity checks the leveling options requires that `GRID_MAX_POINTS_X` and `GRID_MAX_POINTS_Y` be set within certain ranges. So there is no need to define some default here for the case where they might not be set.